resources/views/admin/listCategory.blade.php

```diff
@@ -0,0 +1,102 @@
+@extends('layouts.admin')
+
+
+<!-- Sidebar -->
+@section('sidebar')
+@parent
+
+@endsection
+<!-- End of Sidebar -->
+
+
+@section('content')
+<!-- Begin Page Content -->
+<div class="container-fluid">
+
+    @if (session('createSuccess'))
+            {{-- @include('common.error') --}}
+            <div class="alert alert-success">
+                <i class="fa fa-thumbs-up"></i>
+                <h2>{{session('createSuccess')}}</h2>
+            </div>
+    @endif
+
+
+    <!-- DataTales Table -->
+    <div class="card shadow mb-4">
+        <div class="card-header py-3">
+            <h5 class="m-0 font-weight-bold text-primary">Category List</h5>
+        </div>
+        <div class="card-body">
+            <div class="table-responsive">
+                <a href="{{ route('categories.create') }}" class="btn btn-primary btn-icon-split btn-sm btn-add-new">
+                    <span class="icon">
+                        <i class="fas fa-plus-circle"></i>
+                    </span>
+                    <span class="text">Add new</span>
+                </a>
+                @include('common.checkSave')
+                <table class="table table-bordered" id="dataTable" width="100%" cellspacing="0">
+                    <thead>
+                        <tr>
+                            <th></th>
+                            <th>Name</th>
+                            <th>Actions</th>
+                        </tr>
+                    </thead>
+                    <tfoot>
+                        <tr>
+                            <th></th>
+                            <th>Name</th>
+                            <th>Actions</th>
+                        </tr>
+                    </tfoot>
+                    <tbody>
+                        @foreach ($cat_tours as $cat_tour)
+                        <tr>
+                            <td>{{ $loop->index +1 }}</td>
```

Comment:
biến loop này ở đâu e

Reply:
Dạ biến $loop này mặc định có trong các vòng lặp @foreach hoặc @forelse. Chỗ này em dùng để lấy index của mỗi lần lặp nhằm để in số thứ tự.